lib/screens/settings_screen.dart

```diff
@@ -51,7 +54,46 @@ class _SettingsScreenState extends State<SettingsScreen> {
                 icon: const Icon(Icons.logout_outlined),
                 tooltip: "Log out",
               ),
-            )
+            ),
+             const Center(
+                  child: Text(
+                "contact developer",
+                style: TextStyle(fontSize: 23),
+              )),
+            ListTile(
+              leading: RaisedButton(
+                onPressed: () async =>
+                    await launch("https://wa.me/${"+79871852923"}?text=Hello"),
+                child: const Text('Open WhatsApp'),
+                color: Colors.black,
+              ),
+              trailing: RaisedButton(
+                onPressed: () async =>
+                    await launch("https://t.me/taasneemtoolba"),
+                child: const Text('Open Telegram'),
+                color: Colors.black,
+
+              ),
+            ),
+            ListTile(
+              title: const Text("Rate us"),
+              trailing: RatingBar.builder(
```

Comment:
Do we really need this `RatingBar` package? It's not like the rating actually goes to the Play/App Store. I think it should just be a button that redirects to the relevant store.